phys/module_wind_mav.F

```diff
@@ -1230,11 +1230,12 @@ function find_turb(xc, yc, xt, yt, u, v, sr_angle, sr_dis)  result(ft)
           tmp2 = sign(tmp1,tmp2)
       end if
 
-      angle = real(acos(tmp1/tmp2), kind = 4)
-
-      if (isnan(angle)) then
-         angle = 0.
-      end if
+      tmp1 = tmp1 / tmp2
+      if ( tmp1 < -1 .or. tmp1 > 1 ) then
+        angle = 0
```

Comment:
If we're assigning to a `real`, I think it would be preferable to uses `0.` rather than `0`.